airflow/contrib/example_dags/example_gcp_transfer.py

```diff
@@ -0,0 +1,254 @@
+GCP_TRANSFER_TARGET_BUCKET = os.environ.get('GCP_TRANSFER_TARGET_BUCKET')
+WAIT_FOR_OPERATION_POKE_INTERVAL = os.environ.get('WAIT_FOR_OPERATION_POKE_INTERVAL', 5)
+
+GCP_TRANSFER_SOURCE_AWS_BUCKET = os.environ.get('GCP_TRANSFER_SOURCE_AWS_BUCKET')
+GCP_TRANSFER_FIRST_TARGET_BUCKET = os.environ.get(
+    'GCP_TRANSFER_FIRST_TARGET_BUCKET', 'gcp-transfer-first-target'
+)
+GCP_TRANSFER_SECOND_TARGET_BUCKET = os.environ.get(
+    'GCP_TRANSFER_SECOND_TARGET_BUCKET', 'gcp-transfer-second-target'
+)
+# [END howto_operator_gct_common_variables]
+
+# [START howto_operator_gct_create_job_body_aws]
+create_body_aws = {
+    "description": GCP_DESCRIPTION,
+    "status": GcpTransferJobsStatus.ENABLED,
+    "projectId": GCP_PROJECT_ID,
+    "schedule": {
+        "scheduleStartDate": datetime(2015, 1, 1).date(),
+        "scheduleEndDate": datetime(2030, 1, 1).date(),
+        "startTimeOfDay": datetime.utcnow() + timedelta(minutes=2),
+    },
+    "transferSpec": {
+        'awsS3DataSource': {'bucketName': GCP_TRANSFER_SOURCE_AWS_BUCKET},
+        "gcsDataSink": {"bucketName": GCP_TRANSFER_FIRST_TARGET_BUCKET},
+        "transferOptions": {"overwriteObjectsAlreadyExistingInSink": True},
+    },
+}
+# [END howto_operator_gct_create_job_body_aws]
+
+# [START howto_operator_gct_create_job_body_gcp]
+create_body_gcs = {
+    "description": GCP_DESCRIPTION,
+    "status": GcpTransferJobsStatus.ENABLED,
+    "projectId": GCP_PROJECT_ID,
+    "schedule": {
+        "scheduleStartDate": datetime(2015, 1, 1).date(),
+        "scheduleEndDate": datetime(2030, 1, 1).date(),
+        "startTimeOfDay": datetime.utcnow() + timedelta(minutes=2),
+    },
+    "transferSpec": {
+        'gcsDataSource': {'bucketName': GCP_TRANSFER_FIRST_TARGET_BUCKET},
+        "gcsDataSink": {"bucketName": GCP_TRANSFER_SECOND_TARGET_BUCKET},
+        "transferOptions": {"overwriteObjectsAlreadyExistingInSink": True},
+    },
+}
+# [END howto_operator_gct_create_job_body_gcp]
+
+# [START howto_operator_gct_update_job_body]
+update_body = {
+    "projectId": GCP_PROJECT_ID,
+    "transferJob": {"description": "%s_updated".format(GCP_DESCRIPTION)},
+    "updateTransferJobFieldMask": "description",
+}
+# [END howto_operator_gct_update_job_body]
+
+list_filter_dict = {"projectId": GCP_PROJECT_ID, "jobNames": []}
+
+# [START howto_operator_gct_default_args]
+default_args = {'start_date': days_ago(1)}
+# [END howto_operator_gct_default_args]
+
+with models.DAG(
+    'example_gcp_transfer', default_args=default_args, schedule_interval=None  # Override to match your needs
+) as dag:
+
+    def next_dep(task, prev):
+        prev >> task
+        return task
+
+    # [START howto_operator_gct_create_job]
+    create_transfer_job_from_aws = GcpTransferServiceJobCreateOperator(
+        task_id="create_transfer_job_from_aws", body=create_body_aws
+    )
+    # [END howto_operator_gct_create_job]
+
+    prev_task = create_transfer_job_from_aws
```

Comment:
I also do not like next_dep, it pollutes the example dag. 

But the concept introduced by @sprzedwojski  is very useful during development - where you can essentially comment out several lines of code if you want to exclude some operator from testing (you do not have to manipulate dependencies then).

After looking at your comment and talking to @sprzedwojski  I had an idea that it could be implemented with a simple unary +operator and that it will be super easy. 

I quickly implemented and tested it, and opened a PR for that to Apache: 

https://github.com/apache/airflow/pull/4810

It's a small PR but I am not sure if it will be accepted by community (at the end it adds another way of adding dependencies to a task) - let's hope it will, and then we will be able to use it here as well :).

Reply:
I updated a PR.